packages/angular_devkit/schematics_cli/bin/schematics.ts

```diff
@@ -263,11 +264,46 @@ workflow.execute({
       logger.fatal(err.stack || err.message);
     }
 
-    process.exit(1);
-  },
-  complete() {
-    if (nothingDone) {
-      logger.info('Nothing to be done.');
-    }
-  },
-});
+    return 1;
+  }
+}
+
+ /**
+ * Get usage of the CLI tool.
+ */
+function getUsage(): string {
+  return tags.stripIndent`
+  schematics [CollectionName:]SchematicName [options, ...]
+
+  By default, if the collection name is not specified, use the internal collection provided
+  by the Schematics CLI.
+
+  Options:
+      --debug             Debug mode. This is true by default if the collection is a relative
+                          path (in that case, turn off with --debug=false).
+
+      --allowPrivate      Allow private schematics to be run from the command line. Default to
+                          false.
+
+      --dry-run           Do not output anything, but instead just show what actions would be
+                          performed. Default to true if debug is also true.
+
+      --force             Force overwriting files that would otherwise be an error.
+
+      --list-schematics   List all schematics from the collection, by name. A collection name
+                          should be suffixed by a colon. Example: '@schematics/schematics:'.
```

Comment:
Thank you for this example with the colon! It's not something that would be expected, but this example should allow people to figure it out.